web/src/styles/Aside.css

```diff
@@ -0,0 +1,18 @@
+.aside {
+  position: fixed;
+  top: 35%;
+  width: 100px;
+  left: 3%;
+
+  @media (min-width: 1100px) {
+    left: 10%;
+  }
+
+  &:empty {
+    display: none;
+  }
+
+  &:empty + * {
```

Comment:
❓ hmm what this is for ? and what the adjacent component in this case?

Reply:
Ooooh, good point, the adjacent component was the body, but that's not adjacent anymore!